dlt/destinations/impl/synapse/synapse.py

```diff
@@ -0,0 +1,175 @@
+from typing import ClassVar, Sequence, List, Dict, Any, Optional, cast
+from copy import deepcopy
+from textwrap import dedent
+
+from dlt.common.destination import DestinationCapabilitiesContext
+from dlt.common.destination.reference import SupportsStagingDestination, NewLoadJob
+
+from dlt.common.schema import TTableSchema, TColumnSchema, Schema, TColumnHint
+from dlt.common.schema.typing import TTableSchemaColumns, TTableIndexType
+
+from dlt.destinations.sql_jobs import SqlStagingCopyJob, SqlJobParams
+from dlt.destinations.sql_client import SqlClientBase
+from dlt.destinations.insert_job_client import InsertValuesJobClient
+from dlt.destinations.job_client_impl import SqlJobClientBase
+
+from dlt.destinations.impl.mssql.mssql import (
```

Comment:
@rudolfix is cross-importing between destinations ok for "destination families"?

Reply:
I think we already import duckdb in motherduck. so I say yes